### PR TITLE
Look for .perltidyrc in file and project folder

### PR DESCRIPTION
**Pull request by heinrich-S**
If options.perltidy_profile is not set, search for .perltidyrc in file path and then in project folder.

### What does this implement/fix? Explain your changes.

Trying to find a perltidy profile file at file and project location.

### Does this close any currently open issues?
no

### Any other comments?
copied from PHP-CS-Fixer

### Checklist

Check all those that are applicable and complete.

- [ ] Merged with latest `master` branch
- [ ] Regenerate documentation with `npm run docs`
- [ ] Add change details to `CHANGELOG.md` under "Next" section
- [ ] Added examples for testing to [examples/ directory](examples/)
- [ ] Travis CI passes (Mac support)
- [ ] AppVeyor passes (Windows support)

### CODE REVIEW

**Comment by heinrich-S** [comment]
Can i do something to get this change merged into next release?

File: src/beautifiers/perltidy.coffee [hunk 1]
```diff
@@ -20,7 +21,16 @@ module.exports = class PerlTidy extends Beautifier
...
 
-  beautify: (text, language, options) ->
+  configFiles = ['.perltidyrc']
+  beautify: (text, language, options, context) ->
+    # Find a config file in the working directory if a custom one was not provided
+    if not options.perltidy_profile
+      options.perltidy_profile = if context? and context.filePath? then @findFile(path.dirname(context.filePath), configFiles)
+
+    # Try again to find a config file in the project root
+    if not options.perltidy_profile
```
Comment (Glavin001) [inline src/beautifiers/perltidy.coffee]: I think these `if` statements can be simplified.

```coffeescript
if not options.perltidy_profile
  configRoot = if context? and context.filePath? then path.dirname(context.filePath) else atom.project.getPaths()[0]
  options.perltidy_profile = @findFile(configRoot, configFiles)
```

**Comment by Glavin001** [comment]
Any feedback @stevenzeck ?

**Comment by Glavin001** [comment]
Please let me know when the review requested changes have been addressed / fixed and then this can be reopened. Thank you

**Pull request closed without merging**